pact/message_provider.py

```diff
@@ -115,22 +115,37 @@ def verify(self):
         return_code, _ = verifier.verify_pacts(pact_files, verbose=False)
         assert (return_code == 0), f'Expected returned_code = 0, actual = {return_code}'
 
-    def verify_with_broker(self, enable_pending=False, include_wip_pacts_since=None, **kwargs):
+    def verify_with_broker(
+            self,
+            *pacts,
```

Comment:
Is this a breaking change, because it's a new required positional argument?

Reply:
It still works in the usual way, e.g.
```
verify_with_broker(
  enable_pending=False,
  include_wip_pacts_since=None,
  # other kwargs
)
``

Reply:
Just to demonstrate the idea:

```py
% python
Python 3.11.0 (main, Dec  3 2022, 14:47:23) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def test(*args, **kwargs):
...   print(args, kwargs)
...
>>> test()
() {}
>>>
>>> test('a', 'b', 'c')
('a', 'b', 'c') {}
>>>
>>> test('a', 'b', 'c', foo='bar', baz='buz')
('a', 'b', 'c') {'foo': 'bar', 'baz': 'buz'}
>>>
>>> test(foo='bar', baz='buz')
() {'foo': 'bar', 'baz': 'buz'}
>>>
```

Reply:
Awesome, thanks. TIL